dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/system_config.jsp

```diff
@@ -1,13 +1,24 @@
 <%@ page import="com.liferay.portal.language.LanguageUtil" %>
 <style>
+    :root {
+        --font-size-xxxl: 2rem; /* 32px */
+        --font-size-xxl: 1.75rem; /* 28px */
+        --font-size-xl: 1.5rem; /* 24px */
+        --font-size-lg: 1.25rem; /* 20px */
+        --font-size-default: 14px;
+        --font-size-lmd: 1rem; /* 16px */
+        --font-size-md: var(--font-size-default); /* 14px */
+        --font-size-sm: 0.75rem; /* 12px */
+        --font-size-xs: 0.625rem; /* 10px */
+    }
```

Comment:
If this is not being reused in other files, maybe we can skip it and just add the rem values